docs/dns-client/configuration.md

```diff
@@ -38,6 +38,28 @@ The `cache` object configures caching the results of querying DNS. It has the fo
 
 The `server` object configures the handling of incoming requests. It has the following properties:
 
+  - `bind_retry`: The confguration of the retry mechanism for binding to the listen addresses.
+
+    :::note
+
+    This object is available since `v0.0.3`.
```

Comment:
I'm not sure if we should use monospace for version numbers, really.  @el-termikael, do we have any sort of guidelines about that?

Reply:
I don't think it's necessary